Stellar-contract.x

```diff
@@ -248,7 +248,7 @@ case SCV_LEDGER_KEY_NONCE:
     SCNonceKey nonce_key;
 
 case SCV_CONTRACT_INSTANCE:
-    SCContractInstance instance;
+    SCContractInstance* instance;
```

Comment:
This is a little weird because a null instance is never valid right?

Reply:
Sure, but that's fine. Map and vec are implemented in the same way and this one is not even user-visible.

Reply:
Is this union / optional required because of a cyclic reference or is the value in fact optional?

Reply:
Ah I see this is the same as why Map and Vec are optional.